lanes/mev/mempool.go

```diff
@@ -14,8 +14,8 @@ func TxPriority(config Factory) base.TxPriority[string] {
 	return base.TxPriority[string]{
 		GetTxPriority: func(goCtx context.Context, tx sdk.Tx) string {
 			bidInfo, err := config.GetAuctionBidInfo(tx)
-			if err != nil {
-				panic(err)
+			if err != nil || bidInfo == nil || bidInfo.Bid.IsNil() {
```

Comment:
this is a better worst case in case a faulty tx gets inserted into the MEV lane, gets the lowest priority and will be purged anyways.